tools/test_value_scraper_vmc.py

```diff
@@ -0,0 +1,229 @@
+    config["authorization_host"] = args.authorization_host
+    config["org_id"] = args.org_id
+    config["sddc_id"] = args.sddc_id
+    # TODO will change this when handling of cert generation is done
+    config["verify_ssl"] = False
+
+    config_vcenter["hostname"] = vcenter_server_detail["vcenter_server"]
+    config_vcenter["username"] = vcenter_server_detail["username"]
+    config_vcenter["password"] = vcenter_server_detail["password"]
+    config_vcenter["verify_ssl"] = False
+
+
+def do_it(config_file):
+    try:
+        with config_file.open() as f:
+            config = json.load(f)
+    except Exception as e:
+        exit(f"Bad config: {e}")
+
+    nsx_reverse_proxy_server = get_nsx_reverse_proxy_server(args)
+    print("******** updating vmc nsx config *********")
+    update_vmc_nsx_config(config["vmc_nsx_connect"], nsx_reverse_proxy_server, args)
+    vcenter_server_detail = get_vcenter_server_detail(args)
+    print("******** updating vmc vcenter config *********")
+    update_vmc_vcenter_config(
+        config["vmc_connect"], config["vmc_vcenter_connect"], vcenter_server_detail, args
+    )
+
+    json_config = json.dumps(config, indent=2, sort_keys=True)
+    config_file.write_text(json_config)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Run this before running integration tests for vmc",
+        epilog="""
+        Example  usage creation of file local/vmc_config.json:
+        python tools/test_value_scraper_vmc.py -c local/vmc_config.json -s 1f225622-17ba-4bae-b0ec-a995123a5330 -r <Change Me> -o 10e1092f-51d0-473a-80f8-137652fd0c39
+        """,
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+    parser.add_argument(
+        "-c",
+        dest="create",
+        action="store_true",
+        default=False,
+        help="Create config file if not exists.",
+    )
+    parser.add_argument("CONFIG_FILE", type=Path, help="Path to vmc config file")
+
+    parser.add_argument(
+        "-v",
+        "--vmc_hostname",
+        dest="vmc_hostname",
+        default="stg.skyscraper.vmware.com",
+        help="VMC console",
+    )
+    parser.add_argument(
+        "-a",
+        "--authorization_host",
+        dest="authorization_host",
+        default="console-stg.cloud.vmware.com",
+        help="Hostname of the Cloud Services Platform (CSP)",
+    )
+    parser.add_argument(
+        "-r",
+        "--refresh_key",
+        dest="refresh_key",
+        required=True,
+        help="CSP API token for accessing VMC ",
+    )
+    parser.add_argument("-o", "--org_id", dest="org_id", required=True, help="Organization id")
+    parser.add_argument("-s", "--sddc_id", dest="sddc_id", required=True, help="SDDC id")
+    args = parser.parse_args()
+
+    config_file = args.CONFIG_FILE
+    print(config_file)
+    if not config_file.is_file():
+        if args.create:
+            config_file.write_text(json.dumps(vmc_config_dict))
```

Comment:
**suggestion** either write the values from args to the config file, or interactively get the information from the user.

Reply:
made it like empty template with keys only.